hedera-node/configuration/mainnet/upgrade/feeSchedules.json

```diff
@@ -433,6 +433,52 @@
           ]
         }
       },
+      {
+        "transactionFeeSchedule": {
+          "hederaFunctionality": "TokenAirdrop",
+          "fees": [
+            {
+              "subType": "DEFAULT",
```

Comment:
This should have fees from `TOKEN_FUNGIBLE_COMMON` or `TOKEN_NON_FUNGIBLE_UNIQUE` from CryptoTransfer